prodtests/full-system-test/dpl-workflow.sh

```diff
@@ -140,7 +140,7 @@ else
   if [[ $BEAMTYPE == "pp" ]]; then
     ITS_CONFIG_KEY+="ITSVertexerParam.phiCut=0.5;ITSVertexerParam.clusterContributorsCut=3;ITSVertexerParam.tanLambdaCut=0.2;"
   elif [[ $BEAMTYPE == "PbPb" ]]; then
-    ITS_CONFIG_KEY+="ITSVertexerParam.lowMultBeamDistCut=0;ITSCATrackerParam.nROFsPerIterations=12;ITSCATrackerParam.perPrimaryVertexProcessing=true;"
+    ITS_CONFIG_KEY+="ITSVertexerParam.lowMultBeamDistCut=0;ITSCATrackerParam.nROFsPerIterations=12;"
```

Comment:
You are sure this is correct? This script is the general dpl-workflow script for FST, sync and async reco. If you change something here, it will affect online and offline productions.

Reply:
Yes, indeed I would like not to squash the two commits. I realised that the perPV option was still there for the fst. But this has been disabled in production (see: https://github.com/AliceO2Group/O2DPG/pull/1827 , default is false, I can make it explicit.) .

Reply:
But https://github.com/AliceO2Group/O2DPG/pull/1827 affects only async reco, so in sync reco this setting is currently still present.